packages/babel-plugin-ember-test-metadata/package.json

```diff
@@ -59,5 +59,9 @@
   },
   "publishConfig": {
     "registry": "https://registry.npmjs.org"
+  },
+  "volta": {
+    "node": "12.22.1",
+    "yarn": "1.22.10"
```

Comment:
Should add this at the top level package.json, then `extends`

https://docs.volta.sh/advanced/workspaces

Reply:
Done.